juntagrico/entity/subs.py

```diff
@@ -173,13 +173,21 @@ def next_size_change_date():
 
     @staticmethod
     def get_size_name(types=[]):
-        size_names = []
+        size_names = {}
         for type in types.all():
-            size_names.append(type.__str__())
-        if len(size_names) > 0:
-            return ', '.join(size_names)
-
-        return 'kein Abo'
+            name = type.__str__()
+            if name in size_names:
+                size_names[name] = size_names[name] + 1
+            else:
+                size_names[name] = 1
+
+        if len(size_names) == 0:
+            return _('kein/e/n {0}').format(Config.vocabulary('subscription'))
+
+        formatted = []
+        for key, value in size_names.items():
+            formatted.append(key + ': ' + value.__str__())
+        return '<br>'.join(formatted)
```

Comment:
the <br> tag will probably be filtired oud you need to mark the string as safe
here
https://github.com/juntagrico/juntagrico/blob/master/juntagrico/templates/subscription.html#L59
and here
https://github.com/juntagrico/juntagrico/blob/master/juntagrico/templates/subscription.html#L216
and any other place the method is called,
also make sure it is always called in a template if not make the <br> a parameter so that depending on the place it is called it has the apropriate delimiter

Reply:
https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#safe